src/partials/resource-list/style.scss

```diff
@@ -40,12 +40,9 @@
 
 ul {
   padding: 0;
-  -moz-column-count: 1;
   column-count: 1;
```

Comment:
Let's add autoprefixer in a new patch then, because hand coding all the vendor prefixes is kinda a pain...